signac/contrib/project.py

```diff
@@ -95,6 +95,20 @@ def find_job_ids(self, filter=None, doc_filter=None):
         return self._collection._find(filter)
 
 
+class ProjectConfig(Config):
+    """Extends the project config to make it immutable."""
+    def __init__(self, *args, **kwargs):
+        self._mutable = True
+        super(ProjectConfig, self).__init__(*args, **kwargs)
```

Comment:
Just call the super constructor first and then set the flag.

Reply:
And this is why I shouldn't make PRs late at night. Thanks.